### ISSUE TITLE
ssam TFE green colors

### ISSUE THREAD

**Issue by axet** [ptitSeb/Serious-Engine#19]
Hello!

Your port seems ok, and everything works fine except UI colors. Original games for TFE has green colors, in your build it is BLUE.

**Comment by ptitSeb** [comment]
Do you have a screenshot? I think I have green on my side for TFE.

**Comment by ptitSeb** [comment]
I rechecked, and in-game GUI elements are green on my side. The loading progress bars are blue.

**Comment by axet** [comment]
Yep, that what I meant. Console and loading bar is blue, but it is suppose to be green. Game menu and in-game colors are ok.

**Comment by ptitSeb** [comment]
Ah ok. I'll check that later. I assume thoses are constants, that should be changed for TFE (as the base code is TSE, TFE was added later).

**Comment by axet** [comment]
After clean reinstall: In-game info screen also blue, it suppose to be green. It is probably same issue as for Console colors.
